packages/aws-cdk-lib/aws-ec2/lib/vpc-flow-logs.ts

```diff
@@ -688,7 +711,13 @@ export interface FlowLogOptions {
    * The maximum interval of time during which a flow of packets is captured
    * and aggregated into a flow log record.
    *
-   * @default FlowLogMaxAggregationInterval.TEN_MINUTES
+   * When creating flow logs for a Transit Gateway or Transit Gateway Attachment,
+   * this property must be ONE_MINUTES.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-flowlog.html#cfn-ec2-flowlog-maxaggregationinterval
+   *
+   * @default - FlowLogMaxAggregationInterval.ONE_MINUTES if Transit Gateway or
```

Comment:
Could we update this to say something like `FlowLogMaxAggregationInterval.ONE_MINUTES if creating flow logs for Transit Gateway or....` or `FlowLogMaxAggregationInterval.ONE_MINUTES if target is Transit Gateway or...`? Just saying `if Transit Gateway or Transit Gateway Attachment` doesn't feel very clear.